src/core_seaice/Registry.xml

```diff
@@ -2232,6 +2326,30 @@
 		<var name="iceEnthalpy"			type="real"	dimensions="nIceLayers nCategories nCells Time"		name_in_code="iceEnthalpy"/>
 		<var name="iceSalinity"			type="real"	dimensions="nIceLayers nCategories nCells Time"		name_in_code="iceSalinity"/>
 		<var name="snowEnthalpy"			type="real"	dimensions="nSnowLayers nCategories nCells Time"		name_in_code="snowEnthalpy"/>
+		<var name="snowIceMass"			type="real"	dimensions="nSnowLayers nCategories nCells Time"
```

Comment:
Remove extra spaces between attributes. This is the new style when multiline with descriptions etc.